plugins/brew/brew.plugin.zsh

```diff
@@ -1,4 +1,5 @@
 if (( ! $+commands[brew] )); then
+  local BREW_LOCATION
```

Comment:
As we are going to unset the variable, let's keep that not-local!

Reply:
There was something about old versions of zsh and local variables, now I don't exactly remember what was happening.